File: dice.php
<?php
$count = $_GET["count"];
$sides = $_GET["sides"];

$sum = 0;

for ($i=0; $i < $count; $i++) { 
    $num = rand(1, $sides - 1);
    print_r($num."<br />");
    $sum+=$num;
}

print_r("Sum: ".$sum);

File: dice_cpp/dice.cpp
#include <iostream>
using namespace std;

int main()
{
    int count, sides;
    cout << "Count: ";
    cin >> count;
    cout << "Sides: ";
    cin >> sides;

    srand(time(NULL));

    int sum = 0;

    for (size_t i = 0; i < count; i++)
    {
        int random = 1 + rand() % sides;
        cout << random << "\n";
        sum += random;
    }

    cout << "Sum: " << sum << "\n";
}